calico_mesos/tests/unit/mesos_test.py

```diff
@@ -210,8 +210,8 @@ def side_effect(address, handle_id, attributes, hostname):
             calico_mesos._reserve(hostname, uid, ipv4_addrs, ipv6_addrs)
         self.assertEqual(e.exception.message, "IP '192.168.1.2' already in use.")
 
-        # Test that first IP was unassigned
-        m_datastore.release_ips.assert_called_once_with(ipv4_addrs + ipv6_addrs)
+        # Test that only the first IP was unassigned
+        m_datastore.release_ips.assert_called_once_with(set(["192.168.1.1"]))
```

Comment:
Add another test case (or parameterize the test) so that we throw the other exceptions.